### PR TITLE
KEYCLOAK-2031: Input validation in OTP Policy page fix, test

### PR DESCRIPTION
**Pull request by vramik**

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/console/authentication/OTPPolicyTest.java [hunk 1]
```diff
@@ -66,42 +67,58 @@ public void otpPolicyTest() {
     }      
     
     @Test
+    @Ignore //KEYCLOAK-2051 when you close notification, it is not displayed again
```
Comment (stianst) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/console/authentication/OTPPolicyTest.java]: Is this a mistake?

Comment (vramik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/console/authentication/OTPPolicyTest.java]: No, KEYCLOAK-2051 brokes the test.

Comment (stianst) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/console/authentication/OTPPolicyTest.java]: But, I fixed the tests and they are passing at least on Travis now

Comment (vramik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/console/authentication/OTPPolicyTest.java]: The test is passing on Travis right now but it doesn't catch the bug. So it has to be fixed again. And the fix brings KEYCLOAK-2051.

**Pull request merged**